board-api/src/main/java/com/yapp/crew/controller/BoardController.java

```diff
@@ -0,0 +1,106 @@
+package com.yapp.crew.controller;
+
+import com.yapp.crew.domain.auth.Auth;
+import com.yapp.crew.dto.BoardContentResponseDto;
+import com.yapp.crew.dto.BoardFilterRequestDto;
+import com.yapp.crew.dto.BoardListResponseDto;
+import com.yapp.crew.dto.BoardRequestDto;
+import com.yapp.crew.exception.InternalServerErrorException;
+import com.yapp.crew.model.BoardContentResponseInfo;
+import com.yapp.crew.model.BoardFilter;
+import com.yapp.crew.model.BoardPostRequiredInfo;
+import com.yapp.crew.model.BoardResponseInfo;
+import com.yapp.crew.model.SimpleResponse;
+import com.yapp.crew.service.BoardService;
+import java.util.List;
+import javax.validation.Valid;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.support.PagedListHolder;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@CrossOrigin
+@RestController
+public class BoardController {
+
+  private BoardService boardService;
+  private Auth auth;
+
+  @Autowired
+  public BoardController(BoardService boardService, Auth auth) {
+    this.boardService = boardService;
+    this.auth = auth;
+  }
+
+  @PostMapping(path = "/v1/board/post", produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<?> postBoard(@RequestHeader(value = "Authorization") String token, @RequestBody @Valid BoardRequestDto boardRequestDto) {
+    auth.verifyToken(token);
+    // TODO: try - catch
+    long userId = auth.parseUserIdFromToken(token);
+    BoardPostRequiredInfo boardPostRequiredInfo = BoardPostRequiredInfo.build(boardRequestDto);
+    SimpleResponse simpleResponse = boardService.postBoard(boardPostRequiredInfo, userId);
+
+    return ResponseEntity.ok().body(simpleResponse);
+  }
+
+  @PostMapping(path = "/v1/board/retrieve", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
```suggestion
  @GetMapping(path = "/v1/board", produces = MediaType.APPLICATION_JSON_VALUE)
```
이것으로 수정하시고 위와 경로가 겹쳐서 postmapping 위로 옮기시면 될것 같네요.

Reply:
이거 운동 종목이랑 도시로 게시글 목록 필터링을 해야 해서 request body를 받는데 get 매핑으로도 되는군요..!!